helm/harbor-scanner-trivy/values.yaml

```diff
@@ -33,48 +33,58 @@ securityContext:
   readOnlyRootFilesystem: true
 
 scanner:
+  ## logLevel the log level of `trace`, `debug`, `info`, `warn`, `warning`, `error`, `fatal` or `panic`.
```

Comment:
I'm just curious why there are two `#`.

Reply:
This is just a convention to distinguish comments explaining things from commented out properties. I took this convention from Bitnami's Helm charts:

```yaml
## insecure the flag to skip verifying registry certificate
# insecure: false
```

Otherwise it might be hard to tell what's a description text and what's the property that you can uncomment and set its value.

Reply:
I didn't know it. Thanks!